modules/node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id="node-tuning-hosted-cluster_{context}"]
+= Configuring node tuning in a hosted cluster (Technology Preview)
```

Comment:
No need for (Technology Preview) in the title, as that should be defined by the snippet below.